app/serializers/api/v3/ecf/participant_serializer.rb

```diff
@@ -30,6 +30,11 @@ def eligible_for_funding?(record)
             return false if ecf_participant_eligibility_status == "ineligible"
           end
 
+          def can_view_personal_details?(user, cpd_lead_provider)
+            gdpr_request = user.teacher_profile.gdpr_requests.where(cpd_lead_provider:).first
```

Comment:
this will add n+1 queries, and needs to be added to the main query, happy to amend the queries after discussion